src/app/(admin)/dashboard/_components/DashboardSideBar/DashboardSideBar.tsx

```diff
@@ -0,0 +1,37 @@
+import { X } from 'lucide-react'
+import Image from 'next/image'
+import React from 'react'
+
+export default function DashboardSideBar() {
+    return (
+        <div className='flex-[2] bg-lime-300 h-full w-full'>
+            <div className="contrainer h-full w-[90%] bg-red-300 m-auto pt-5">
+                <div className="logo">
+                    <Image src={`/logo/logo.svg`} width={150} height={150} alt="" />
```

Comment:
Please make the image import consistent as you have in 
 **src/app/(admin)/(auth)/_components/AuthCarousel.tsx**